src/ansys/motorcad/core/geometry.py

```diff
@@ -102,29 +103,49 @@ def remove_entity(self, entity_remove):
                         self.entities.remove(entity)
 
     # method to receive region from Motor-CAD and create python object
-    def _from_json(self, json):
+    @classmethod
+    def _from_json(cls, json, motorcad_instance=None):
         """Convert class from json object.
 
         Parameters
         ----------
         json: dict
             Represents geometry region
+        motorcad_instance : ansys.motorcad.core.MotorCAD
+            Motor-CAD instance that is connected
```

Comment:
```suggestion
            Motor-CAD instance to connect to. The default is ``None``.
```
What behavior occurs when ``None`` is used?